connectomics/discfibers/ea_discfibers_merge_pathways.m

```diff
@@ -134,8 +160,11 @@
         ftr2.idx = cat(1, C_fibState_idx{:});
         
         % store as fiberActivation_side.mat in the corresp. stim folder
-        [filepath,name,ext] = fileparts(pam_file);
-        fiberActivation_merged = [filepath,filesep,'fiberActivation_',side_name,'.mat'];
+        [filepath,~,~] = fileparts(pam_file);
+        %BIDS notation
+        [~,subj_tag,~] = fileparts(obj.M.patient.list{sub});
+        subSimPrefix = [subj_tag, '_sim-'];
+        fiberActivation_merged = [filepath,filesep,subSimPrefix,'fiberActivation_',side_name,'.mat'];
```

Comment:
the naming is not correct.  should contain `sim-fiberActivation_model-ossdbs_hemi-[LR].mat`